mix.exs

```diff
@@ -9,11 +9,11 @@ defmodule ValidField.Mixfile do
      elixirc_paths: elixirc_paths(Mix.env),
      build_embedded: Mix.env == :prod,
      start_permanent: Mix.env == :prod,
-     deps: deps,
+     deps: deps(),
```

Comment:
oh awesome, I'm very happy to see Elixir is now warning on empty method calls :)